resources/test-config.edn

```diff
@@ -6,8 +6,9 @@
  :peer-config
  {:zookeeper/address "127.0.0.1:2188"
   :onyx.peer/job-scheduler :onyx.job-scheduler/greedy
-  :onyx.messaging/impl :aeron
-  :onyx.messaging/ip "localhost"}
+  :onyx.messaging/config {:messaging/impl :aeron
```

Comment:
No problem. I was just trying to distinguish between machine config options and cluster options, but you're right.